src/WebSocketsClient.cpp

```diff
@@ -117,7 +133,9 @@ void WebSocketsClient::loop(void) {
                 _client.tcp = NULL;
             }
             _client.ssl = new WiFiClientSecure();
+#if !defined(ESP32)
             _client.tcp = _client.ssl;

```

Comment:
if 80 is used 8080 is good choice, 81 where working fine too.
Firefox bad port list,
https://dxr.mozilla.org/mozilla-central/source/netwerk/base/nsIOService.cpp#93

may this changes here:
https://github.com/Links2004/arduinoWebSockets/compare/write_big_data
help for the writing.